index.html

```diff
@@ -1682,6 +1682,39 @@ <h3>Context Injection</h3>
           </p>
         </section>
 
+        <section>
+          <h3>Securing Data Losslessly</h3>
+
+          <p>
+HTML processors are designed to continue processing if recoverable errors are
+detected. JSON-LD processors operate in a similar manner. This design philosophy
+was used to ensure that developers need only use the parts of the JSON-LD
+language that they find useful to them without causing the processor to throw
+errors on things that might not be important to the developer. For example,
+JSON-LD processors are designed to not throw errors, but rather warn developers,
+when things such as undefined terms are encountered.
+          </p>
+
+          <p>
+When converting from JSON-LD to an RDF Dataset, such as when canonicalizing a
+document [[?RDF-CANON]], undefined terms can be dropped silently. The dropping
+of undefined terms results in all data associated with the undefined term not
+being protected by a digital proof. This creates a mismatch of expectations,
+where a developer who is unaware of how a JSON-LD processor works, would think
+that certain data was being secured and would be surprised to find out that it
+was not and no error was thrown. This specification requires that any
+recoverable loss of data when performing JSON-LD processing results in an error
+to avoid a mismatch in security expectations from developers.
```

Comment:
```suggestion
document [[?RDF-CANON]], undefined terms and relative URLs can be dropped
silently. When values are dropped, they are not protected by a digital proof. This
creates a mismatch of expectations, where a developer who is unaware of how
a JSON-LD processor works, might think that certain data was being secured
and would be surprised to find out that it was not and no error was thrown.
This specification requires that any recoverable loss of data when
performing JSON-LD transformations results in an error to avoid a mismatch
in security expectations from developers.
```

Reply:
A few nits in @dlongley's suggestion...
```suggestion
document [[?RDF-CANON]], undefined terms and relative URLs can be dropped
silently. When values are dropped, they are not protected by a digital proof. This
creates a mismatch of expectations, where a developer, who is unaware of how
a JSON-LD processor works, might think that certain data was being secured,
and would be surprised to find out that it was not when no error was thrown.
This specification requires that any recoverable loss of data when
performing JSON-LD transformations results in an error to avoid a mismatch
in security expectations of developers.
```